src/client/Client.js

```diff
@@ -338,6 +339,19 @@ class Client extends BaseClient {
       .then(app => new ClientApplication(this, app));
   }
 
+  /**
+   * Tries to fetch the specified guild, if available to the "preview" endpoint.
```

Comment:
To keep it consistent with other methods here.
```suggestion
   * Obtains a guild preview from Discord, only available for "public" guilds.
```